ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/ApiScenarioTransform.scala

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.infrastructure
+
+import com.digitalasset.daml.lf.data.Ref.{ChoiceName, ContractIdString, PackageId, QualifiedName}
+import com.digitalasset.daml.lf.data.{BackStack, ImmArray, Ref}
+import com.digitalasset.daml.lf.language.Ast
+import com.digitalasset.daml.lf.transaction.Node.KeyWithMaintainers
+import com.digitalasset.daml.lf.value.Value.{AbsoluteContractId, VersionedValue}
+import com.digitalasset.daml.lf.value.{Value, ValueVersions}
+import com.digitalasset.ledger.api.v1.event.{
+  CreatedEvent => ApiCreatedEvent,
+  ExercisedEvent => ApiExercisedEvent
+}
+import com.digitalasset.ledger.api.v1.transaction.{TransactionTree, TreeEvent}
+import com.digitalasset.ledger.api.v1.value.{Record, Value => ApiValue}
+import com.digitalasset.ledger.api.validation.ValueValidator
+import com.digitalasset.platform.common.{PlatformTypes => P}
+import com.digitalasset.platform.server.api.validation.ErrorFactories
+import io.grpc.StatusRuntimeException
+import scalaz.Traverse
+import scalaz.std.either._
+import scalaz.std.list._
+
+@SuppressWarnings(
+  Array(
+    "org.wartremover.warts.Any"
+  ))
+class ApiScenarioTransform(ledgerId: String, packages: Map[Ref.PackageId, Ast.Package])
```

Comment:
Is the intention to remove one of the copies of this file?

Reply:
The copy has been already remove from the integration tests because the semantic tests have been already ~eradicated~ removed from there as well.

Reply:
Uh, what I just said is wrong. Removing the original as of know. Good catch, thanks!

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2282/commits/af4baca1712a9dc00000b388d78a82f6f755d6fa